pkg/model/config/config.go

```diff
@@ -93,23 +93,25 @@ func (c Config) HasResource(target GVK) bool {
 	return false
 }
```

Comment:
we need to cover it with unit tests: https://coveralls.io/builds/34399100/source?filename=pkg/model/config/config.go

Reply:
Done